ivy/__init__.py

```diff
@@ -1155,3 +1155,14 @@ def __exit__(self, type, value, traceback):
 
 def dynamic_backend_as(value):
     return DynamicBackendContext(value)
+
+#We monkey patch np.array to enable ivy.Array conversion with tensorflow backend
+_np_array = np.array
+
+@wraps(_np_array)
+def nparray(*args, **kwargs):
+    if isinstance(args[0], ivy.Array):
+        return ivy.to_numpy(*args, **kwargs)
+    return  _np_array(*args, **kwargs)
+
+np.array = nparray
```

Comment:
we shouldn't override the whole `np.array` I guess :sweat_smile:

Reply:
Sure will have look. I guess the most granular method is `np.core.multiarray.array(*args, **kwargs)`